### PR TITLE
Sidebar Form Skeleton w/API request

### PR DESCRIPTION
**Pull request by Sensationull**

### CODE REVIEW

File: client/src/index.js [hunk 1]
```diff
@@ -1,18 +1,36 @@
...
     this.state = {
       title: ''
     };
+    this.asyncGet = this.asyncGet.bind(this);
   }
 
+  asyncGet(callback) {
+    axios.get('/api/buytest')
+      .then(function (response) {
+        // console.log('THA RESPONSE', response);
```
Comment (Ben-Mustion) [inline client/src/index.js:18]: You can remove the console log here and on line 22

File: client/src/components/WatchList.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    return (
+      <button>Remove From Watchlist</button>
+    );
+  }
+}
+
+export default WatchList;
+
+// this.state is going to be conditionally rendered, changing the button from 'Add to Watchlist' to 'Remove From Watchlist' on click
+// it needs a function and a value for add and a value for remove.
```
Comment (Ben-Mustion) [inline client/src/components/WatchList.jsx:24]: You should add a blank line at the end of each component for styling purposes.

**Review by Ben-Mustion — CHANGES_REQUESTED** [review]

File: client/src/index.js (hunk 1 above)
Comment (Sensationull) [reply client/src/index.js:18]: Thanks, I'll make the changes

File: client/src/components/WatchList.jsx (hunk 2 above)
Comment (Sensationull) [reply client/src/components/WatchList.jsx:24]: I'll get to it

**Pull request merged**